volcli/commands.go

```diff
@@ -89,7 +89,7 @@ var Commands = []cli.Command{
 					Usage: "Provide key=value options to create the volume",
```

Comment:
No.

On 27 Jun 2016, at 13:18, unclejack wrote:

>> @@ -89,7 +89,7 @@ var Commands = []cli.Command{
>>  					Usage: "Provide key=value options to create the volume",
>
> Are there any use commands we should be getting rid of now?
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/volplugin/pull/313/files/a4dd7184906418e9c3fa0abbbfa0cad298fd36f4#r68647891